sklearn/ensemble/_gb.py

```diff
@@ -463,8 +459,12 @@ def fit(self, X, y, sample_weight=None, monitor=None):
             Fitted estimator.
         """
         if self.criterion in ("absolute_error", "mae"):
```

Comment:
Could we do:
if criterion not in []:
    raise general errir message

In particular, remove the „anymore“.